generators/server/templates/src/main/java/package/domain/_User.java

```diff
@@ -28,7 +28,7 @@
 import java.time.ZonedDateTime;<% } %>
 
 /**
- * A user.
+ * An user.
```

Comment:
`An` should be `A` here.  

When `u` makes the same sound as the `y` in `you`, or `o` makes the same sound as `w` in `won`, then `a` is used.